### ISSUE TITLE
Can't start standalone jar

### ISSUE THREAD

**Issue by BlackYps** [Deep-Symmetry/afterglow#71]
**Describe the bug**
When trying to run the jar package with `java -jar afterglow.jar` I get the error below. The documentation doesn't mention any specific java requirements, so I don't know what I could do to fix this. 

**Desktop (please complete the following information):**
 - OS: Linux (Manjaro stable)
 - Java Version: 
 openjdk 11.0.13 2021-10-19
OpenJDK Runtime Environment (build 11.0.13+8)
 - Project Version 0.2.4

**Additional context**
```
Exception in thread "main" java.lang.ExceptionInInitializerError
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:398)
        at clojure.lang.RT.classForName(RT.java:2168)
        at clojure.lang.RT.classForName(RT.java:2177)
        at clojure.lang.RT.loadClassForName(RT.java:2196)
        at clojure.lang.RT.load(RT.java:443)
        at clojure.lang.RT.load(RT.java:419)
        at clojure.core$load$fn__5677.invoke(core.clj:5893)
        at clojure.core$load.invokeStatic(core.clj:5892)
        at clojure.core$load.doInvoke(core.clj:5876)
        at clojure.lang.RestFn.invoke(RestFn.java:408)
        at clojure.core$load_one.invokeStatic(core.clj:5697)
        at clojure.core$load_one.invoke(core.clj:5692)
        at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
        at clojure.core$load_lib.invokeStatic(core.clj:5736)
        at clojure.core$load_lib.doInvoke(core.clj:5717)
        at clojure.lang.RestFn.applyTo(RestFn.java:142)
        at clojure.core$apply.invokeStatic(core.clj:648)
        at clojure.core$load_libs.invokeStatic(core.clj:5774)
        at clojure.core$load_libs.doInvoke(core.clj:5758)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:648)
        at clojure.core$require.invokeStatic(core.clj:5796)
        at clojure.core$require.doInvoke(core.clj:5796)
        at clojure.lang.RestFn.invoke(RestFn.java:3894)
        at afterglow.web.routes.show_control$loading__5569__auto____23876.invoke(show_control.clj:1)
        at afterglow.web.routes.show_control__init.load(Unknown Source)
        at afterglow.web.routes.show_control__init.<clinit>(Unknown Source)
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:398)
        at clojure.lang.RT.classForName(RT.java:2168)
        at clojure.lang.RT.classForName(RT.java:2177)
        at clojure.lang.RT.loadClassForName(RT.java:2196)
        at clojure.lang.RT.load(RT.java:443)
        at clojure.lang.RT.load(RT.java:419)
        at clojure.core$load$fn__5677.invoke(core.clj:5893)
        at clojure.core$load.invokeStatic(core.clj:5892)
        at clojure.core$load.doInvoke(core.clj:5876)
        at clojure.lang.RestFn.invoke(RestFn.java:408)
        at clojure.core$load_one.invokeStatic(core.clj:5697)
        at clojure.core$load_one.invoke(core.clj:5692)
        at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
        at clojure.core$load_lib.invokeStatic(core.clj:5736)
        at clojure.core$load_lib.doInvoke(core.clj:5717)
        at clojure.lang.RestFn.applyTo(RestFn.java:142)
        at clojure.core$apply.invokeStatic(core.clj:648)
        at clojure.core$load_libs.invokeStatic(core.clj:5774)
        at clojure.core$load_libs.doInvoke(core.clj:5758)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:648)
        at clojure.core$require.invokeStatic(core.clj:5796)
        at clojure.core$require.doInvoke(core.clj:5796)
        at clojure.lang.RestFn.invoke(RestFn.java:805)
        at afterglow.web.routes.home$loading__5569__auto____23874.invoke(home.clj:1)
        at afterglow.web.routes.home__init.load(Unknown Source)
        at afterglow.web.routes.home__init.<clinit>(Unknown Source)
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:398)
        at clojure.lang.RT.classForName(RT.java:2168)
        at clojure.lang.RT.classForName(RT.java:2177)
        at clojure.lang.RT.loadClassForName(RT.java:2196)
        at clojure.lang.RT.load(RT.java:443)
        at clojure.lang.RT.load(RT.java:419)
        at clojure.core$load$fn__5677.invoke(core.clj:5893)
        at clojure.core$load.invokeStatic(core.clj:5892)
        at clojure.core$load.doInvoke(core.clj:5876)
        at clojure.lang.RestFn.invoke(RestFn.java:408)
        at clojure.core$load_one.invokeStatic(core.clj:5697)
        at clojure.core$load_one.invoke(core.clj:5692)
        at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
        at clojure.core$load_lib.invokeStatic(core.clj:5736)
        at clojure.core$load_lib.doInvoke(core.clj:5717)
        at clojure.lang.RestFn.applyTo(RestFn.java:142)
        at clojure.core$apply.invokeStatic(core.clj:648)
        at clojure.core$load_libs.invokeStatic(core.clj:5774)
        at clojure.core$load_libs.doInvoke(core.clj:5758)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:648)
        at clojure.core$require.invokeStatic(core.clj:5796)
        at clojure.core$require.doInvoke(core.clj:5796)
        at clojure.lang.RestFn.invoke(RestFn.java:457)
        at afterglow.web.handler$loading__5569__auto____19946.invoke(handler.clj:1)
        at afterglow.web.handler__init.load(Unknown Source)
        at afterglow.web.handler__init.<clinit>(Unknown Source)
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:398)
        at clojure.lang.RT.classForName(RT.java:2168)
        at clojure.lang.RT.classForName(RT.java:2177)
        at clojure.lang.RT.loadClassForName(RT.java:2196)
        at clojure.lang.RT.load(RT.java:443)
        at clojure.lang.RT.load(RT.java:419)
        at clojure.core$load$fn__5677.invoke(core.clj:5893)
        at clojure.core$load.invokeStatic(core.clj:5892)
        at clojure.core$load.doInvoke(core.clj:5876)
        at clojure.lang.RestFn.invoke(RestFn.java:408)
        at clojure.core$load_one.invokeStatic(core.clj:5697)
        at clojure.core$load_one.invoke(core.clj:5692)
        at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
        at clojure.core$load_lib.invokeStatic(core.clj:5736)
        at clojure.core$load_lib.doInvoke(core.clj:5717)
        at clojure.lang.RestFn.applyTo(RestFn.java:142)
        at clojure.core$apply.invokeStatic(core.clj:648)
        at clojure.core$load_libs.invokeStatic(core.clj:5774)
        at clojure.core$load_libs.doInvoke(core.clj:5758)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:648)
        at clojure.core$require.invokeStatic(core.clj:5796)
        at clojure.core$require.doInvoke(core.clj:5796)
        at clojure.lang.RestFn.invoke(RestFn.java:1789)
        at afterglow.core$loading__5569__auto____19940.invoke(core.clj:1)
        at afterglow.core__init.load(Unknown Source)
        at afterglow.core__init.<clinit>(Unknown Source)
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:398)
        at clojure.lang.RT.classForName(RT.java:2168)
        at clojure.lang.RT.classForName(RT.java:2177)
        at clojure.lang.RT.loadClassForName(RT.java:2196)
        at clojure.lang.RT.load(RT.java:443)
        at clojure.lang.RT.load(RT.java:419)
        at clojure.core$load$fn__5677.invoke(core.clj:5893)
        at clojure.core$load.invokeStatic(core.clj:5892)
        at clojure.core$load.doInvoke(core.clj:5876)
        at clojure.lang.RestFn.invoke(RestFn.java:408)
        at clojure.lang.Var.invoke(Var.java:379)
        at clojure.lang.Util.loadWithClass(Util.java:250)
        at afterglow.core.<clinit>(Unknown Source)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.DatatypeConverter
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:398)
        at clojure.lang.RT.classForName(RT.java:2168)
        at clojure.lang.RT.classForNameNonLoading(RT.java:2181)
        at org.httpkit.server$loading__5569__auto____24166.invoke(server.clj:1)
        at org.httpkit.server__init.load(Unknown Source)
        at org.httpkit.server__init.<clinit>(Unknown Source)
        ... 126 more
```

**Comment by brunchboy** [comment]
Hi! Sorry you ran into this, it’s due to the fact that I haven’t been working on Afterglow lately, all my open-source energy is being taken up by Beat Link Trigger, but the problem is that Java 9 and later don’t include classes that Clojure expects to find. Here is a Stack Overflow article that may help you get past this until I can find time to start building the Afterglow jars with a more recent Java SDK. https://stackoverflow.com/questions/43574426/how-to-resolve-java-lang-noclassdeffounderror-javax-xml-bind-jaxbexception

**Comment by brunchboy** [comment]
Also, if you are willing to get Afterglow building and running with a current LTS Java version, a pull request would be most welcome!

**Comment by brunchboy** [comment]
Strangely, I just tried building the jar from source using the latest code on the master branch, and it built and ran fine using:
```
openjdk version "11.0.5" 2019-10-15 LTS
OpenJDK Runtime Environment Corretto-11.0.5.10.2 (build 11.0.5+10-LTS)
OpenJDK 64-Bit Server VM Corretto-11.0.5.10.2 (build 11.0.5+10-LTS, mixed mode)
```
So I am less sure what you are running into. You might want to try that?

**Comment by brunchboy** [comment]
Actually, @BlackYps, I have created a preview release containing this jar which I just built using OpenJDK 11. Could you please try it and see if it works better for you? If it does, that might be reason enough to promote it to an actual release.
https://github.com/Deep-Symmetry/afterglow/releases/tag/v0.2.5-SNAPSHOT

**Comment by BlackYps** [comment]
Yeah, building from source would have been my next step to see how I can get it working. Your new release works for me. Building from source should then probably also work for me. I agree that it is a good idea to make this an official release. Makes the project seem less abandoned and it provides a more "out of the box" experience.

**Comment by brunchboy** [comment]
Oh, absolutely I want the released jar to work on current Java releases! It has been a long time since Java 8 went out of support. Ok, I have published release 0.2.5, which also includes updating a bunch of dependencies, and all that work I did but never released a few years ago. Thanks again for calling this issue to my attention, and good luck with the steep learning curve of Afterglow, even when the jar launches properly! 😄 

Please let me know if you run into any other problems.

**Comment by BlackYps** [comment]
Thank you, I already read some of the documentation and it is pretty daunting, mainly because I don't know any clojure and it is so different from the languages I know. On the other hand I absolutely want to use Afterglow, because it seems so incredibly powerful and capable of what I want to achieve.
I have in fact already run into the next problem, I will open a new issue for that.
